src/main/java/org/team340/robot/subsystems/GooseneckRollers.java

```diff
@@ -0,0 +1,70 @@
+package org.team340.robot.subsystems;
+
+import com.ctre.phoenix6.hardware.CANdi;
+import com.ctre.phoenix6.hardware.TalonFXS;
+import edu.wpi.first.wpilibj2.command.Command;
+
+import static edu.wpi.first.wpilibj2.command.Commands.*;
+
+import java.util.function.DoubleSupplier;
+import org.team340.lib.util.command.GRRSubsystem;
+import org.team340.robot.Constants.RobotMap;
+
```

Comment:
```suggestion

@Logged
```
This allows the annotation processor to generate a logging file for the subsystem